backend/routes/api.php

```diff
@@ -34,6 +34,6 @@
     Route::apiResource('auth', AuthController::class)->only('update')->parameters(['auth' => 'user']);
     Route::apiResource('users.accounts', AccountController::class)->shallow()->only(['index']);
     Route::apiResource('accounts', AccountController::class)->only(['index', 'show']);
-    Route::apiResource('accounts.transactions', TransactionController::class)->shallow()->only(['index']);
+    Route::apiResource('accounts.transactions', TransactionController::class)->shallow()->only(['index', 'store'])->parameters(['accounts' => 'myAccount']);
```

Comment:
remove parameter chain
no need to change the name of the parameter here, 

also shouldnt the parameter be `account` for this route?

`account` is already sensible, i dont see the need to change the term

![image](https://github.com/framgia/sph-flutter/assets/95029803/a7e84fd7-0525-4a0a-80c4-4769480c5e31)